src/fenicsxconcrete/experimental_setup/am_multiple_layers.py

```diff
@@ -52,6 +46,7 @@ def default_parameters() -> dict[str, pint.Quantity]:
         """
 
         setup_parameters = {}
+        setup_parameters["degree"] = 2 * ureg("")  # polynomial degree
```

Comment:
I would remove the polynomial degree from all the experiments and put it into the finitem element problems. One could argue, that for isoparametric elements, this would belong in the mesh creation, but when do we actually do this?

Reply:
For the compression_cylinder experiment the degree is actually used for generating the mesh with gmsh
In all other cases I moved degree to the material problem

Reply:
In that case, I would say that we differentiate between polynomial degree and element geometry degree. So in the experiment, we could have `element_order` and in the finite element problem we have `degree` as parameter.

Reply:
done